### DISCUSSION TITLE
Add GeoLib for geodesic functions

### DISCUSSION THREAD

**Discussion by sindizzy** [DotSpatial/DotSpatial#1454] [Ideas]
Unless I am not aware of it there are no geodesic functions in DotSpatial and/or NTS. I have a project that uses DotSpatial and I have added also the GeoLib library which allows me to generate geodesic paths between two points. Is that something that can be looked at?

**Comment by jany-tenaj** [comment]
Can't say I'd need it or know what it does. But if you could add it as a plugin whoever needs it could use it?

**Comment by sindizzy** [comment]
It lets you generate what most folks refer to as "great circle paths" between two points. I don't believe NTS has these functions. Let me check on that.
